.github/workflows/collab-guard.yml

```
@@ -5,6 +5,58 @@
+      - name: Add needs-split label for large PRs
+        if: steps.pr_size.outputs.is_large == 'true'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              labels: ['needs-split']
+            })
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)